prowler/providers/azure/services/iam/iam_service.py

```diff
@@ -19,10 +19,10 @@ def __get_roles__(self):
         for subscription, client in self.clients.items():
             try:
                 roles.update({subscription: []})
-                for role in client.role_definitions.list(
+                all_roles = client.role_definitions.list(
```

Comment:
Maybe here you can build two objects, `custom_roles` and `managed_roles` (or whatever is named by Azure).

Reply:
Done! custom_roles for CustomRoles and builtin_roles for Built-In Roles